cis/Makefile

```diff
@@ -15,6 +26,12 @@ $(targets): target/%: % target/configuration.txz
 
 configuration: target/configuration.txz
 
+pdfs:
+	mkdir pdfs
+	$(GET) pdfs/cis_debian9.pdf https://publisher.normation.com/build-dependencies/cis_debian9.pdf $(DEBIAN9_SHA256)
+	$(GET) pdfs/cis_redhat7.pdf https://publisher.normation.com/build-dependencies/cis_redhat7.pdf $(REDHAT7_SHA256)
```

Comment:
could you put them in a `cis` folder? We'll probably have other dependencies there.

Reply:
fixed